BrightID/src/components/PendingConnectionsScreens/actions/channelThunks.js

```diff
@@ -189,6 +201,13 @@ export const fetchChannelProfiles = createAsyncThunk(
   async (channelId, { getState, dispatch }) => {
     const channel = selectChannelById(getState(), channelId);
     let profileIds = await channel.api.list(channelId);
+
+    // ignore channelInfo.json
```

Comment:
`// ignore channelInfo.json`

Could you please explain this comment?

Reply:
Comment updated